spark/src/test/scala/org/apache/spark/sql/delta/DeltaUpdateCatalogSuite.scala

```diff
@@ -410,70 +410,6 @@ class DeltaUpdateCatalogSuite
     }
   }
 
-  test("Very long schemas can be stored in the catalog - " +
```

Comment:
you can just `ignore` right?

Reply:
+1 - also add a comment that they are disabled temporarily because they are flaky and needs to be fixed.